artifactory/commands/repository/repository.go

```diff
@@ -36,51 +39,133 @@ func (rc *RepoCommand) TemplatePath() string {
 	return rc.templatePath
 }
 
+type repoCreateHandler interface {
```

Comment:
Earlier we renamed it as repoCreateHandler from handler, but if this interface is applicable for both create/update lets rename it as `repoCreateUpdateHandler` or `repoUpsertHandler` or anthing else which fits better